.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+Documentation:
+  Enabled: false
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
It was a little confusing the first time we used Hound CI and were used to your style. Ultimately bought into their thinking behind it and haven't looked back.